versioned_docs/version-2.6/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -2,13 +2,18 @@
 title: Configure Azure AD
 ---
 
-<Tabs>
-  <TabItem value="Rancher v2.6.7">
+import Tabs from '@theme/Tabs';
+ import TabItem from '@theme/TabItem';
+
+ <Tabs>
+   <TabItem value="Rancher v2.6.7">
 
 ## Microsoft Graph API
 
 Microsoft Graph API is now the flow through which you will set up Azure AD. The below sections will assist [new users](#new-user-setup) in configuring Azure AD with a new instance as well as assist existing Azure app owners in [migrating to the new flow](#migrating-from-azure-ad-graph-api-to-microsoft-graph-api).
 
+The new Microsoft Graph API flow in Rancher is constantly evolving. We recommend users to be on the latest version of 2.6, which will keep receiving new features and fixes.
```

Comment:
Will adjust, thanks! The reason I am adding this is because this page if for v2.6.7+. But there was some behavior in v2.6.7-v2.6.10 that we plan to update in v2.6.11. That's why we encourage users to upgrade to just to v2.6.7, but the latest if possible.

Reply:
Yeah, as I got deeper into the review it became clear that we have to retain some references to time if only for the deadline. Sometimes it's just unavoidable.